module/data/abstract.mjs

```diff
@@ -80,7 +88,7 @@ export default class SystemDataModel extends foundry.abstract.DataModel {
   static mixin(...templates) {
```

Comment:
We should annotate the return value here as `{typeof SystemDataModel}` to indicate that we are returning the constructor, not an instance.